test/test_common/utility.h

```diff
@@ -26,6 +29,16 @@ namespace Envoy {
     EXPECT_EQ(message, std::string(e.what()));                                                     \
   }
 
+class TestRandomGenerator {
```

Comment:
Not currently.  It would require faking out the uuid string which no one needs.    No callers actually want the RandomGenerator interface they just want a random  number.
I can of course do it anyway if we anticipate the use case.